src/app/providers/NFT/BoosterContext.tsx

```diff
@@ -52,7 +52,8 @@ export const BoosterProvider = ({ children }: BoosterContextProviderProps) => {
       return { boostData, isLoading, error, hasActiveCampaign }
     }
 
-    const currentBoost = boostData?.holders[address]
+    const [, currentBoost] =
+      Object.entries(boostData?.holders ?? {}).find(([key]) => isAddressEqual(key as Address, address)) ?? []
```

Comment:
Here, I don't see the necessity of converting everything to lowercase, if we have 1k holders that means 1k lowercase.